.github/workflows/on-push-workflow.yml

```diff
@@ -57,6 +58,7 @@ jobs:
           echo "STMFetchUpdateStatic=$STMFetchUpdateStatic" >> $GITHUB_ENV
           echo "STMFilterDailyStatic=$STMFilterDailyStatic" >> $GITHUB_ENV
           echo "STMMergeDailyVehiclePosition=$STMMergeDailyVehiclePosition" >> $GITHUB_ENV
+          echo "STMAnalyzeSegments=$STMAnalyzeSegments" >> $GITHUB_ENV
 
```

Comment:
Rebase on the main branch. There seem to be missing lines. Expecially this one:
`UpdateCheck="${BIXI},${STMAnalyzeDailyStop},${STMCreateDailyStop},${STMFetchTripUpdates},${STMFetchVehiclePositions},${STMFetchUpdateStatic},${STMFilterDailyStatic},${STMMergeDailyVehiclePosition}"
`
that you will have to update.

Reply:
They are not missing only hidden because he didn't edit them.
So @maximeBAY don't forget to add your variable at the end of line 70 (the same line that Guy referenced in his comment)

Reply:
added variable at line 70